docs/guides/contribute/local_docs.it.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Documentazione Locale - Veloce
+author: Lukas Magauer
+contributors: Steven Spencer, Franco Colussi
+tested with: 8.6, 9.0
+tags:
+  - documentation
+  - local server
+---
+
+# Introduzione
+
+Puoi costruire il sistema di documentazione localmente senza Docker o LXD se vuoi. Se si sceglie di usare questa procedura, tuttavia, è bene tenere presente che se si si scrive molto codice in Python o si usa Python localmente, la scelta più sicura è quella di creare un ambiente virtuale Python [descritto qui](https://docs.python.org/3/library/venv.html). Questo mantiene tutti i processi Python protetti gli uni dagli altri, che è raccomandato. Se scegliete di usare questa procedura senza l'ambiente virtuale Python, sappiate che state correndo un certo rischio.
```

Comment:
?

Reply:
Thanks @SergeCroise. Here's something to keep in mind: The Crowdin interface is used to translate the source (in this case English) document into another language. If something is not clear within a language, then it probably needs to be addressed within the source document, which in this case is [here](https://docs.rockylinux.org/guides/contribute/local_docs/) It can't be done outside of the Crowdin interface, except by first editing the source document. Hopefully, I've defined that so that you can understand, but if not, please let me know!

If something isn't clear, I'd love for you to push a change to that source document and then it will automatically get re-translated into the other languages over time.  Let me know!